uPortal-webapp/src/main/webapp/scripts/jstree/jstree.js

```diff
@@ -1,18 +1,16 @@
 /*globals jQuery, define, exports, require, window, document, postMessage */
-(function (factory) {
-	"use strict";
-	if (typeof define === 'function' && define.amd) {
-		define(['jquery'], factory);
-	}
-	else if(typeof exports === 'object') {
-		factory(require('jquery'));
-	}
-	else {
-		factory(jQuery);
-	}
-}(function ($, undefined) {
-	"use strict";
-/*!
+;(function(factory) {
+  'use strict'
+  if (typeof define === 'function' && define.amd) {
+    define(['jquery'], factory)
+  } else if (typeof exports === 'object') {
+    factory(require('jquery'))
+  } else {
+    factory(jQuery)
+  }
+})(function($, undefined) {
+  'use strict'
+  /*!
  * jsTree 3.1.1
```

Comment:
A 3rd-party file with a ton text changes, but no version change???  (It doesn't add up.)

Reply:
This is because the file was formatted.